src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -157,6 +157,7 @@ CA2246 | Usage | Info | AssigningSymbolAndItsMemberInSameStatement, [Documentati
 CA2247 | Usage | Warning | DoNotCreateTaskCompletionSourceWithWrongArguments, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2247)
 CA2248 | Usage | Info | ProvideCorrectArgumentToEnumHasFlag, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2248)
 CA2249 | Usage | Info | PreferStringContainsOverIndexOfAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2249)
+CA2250 | Usage | Warning | DetectPreviewFeatureAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2250)
```

Comment:
Please do no hand edit shipped releases file, this is the reason that the code fix added to Changed Rules table in the unshipped file. This file should be reverted.

Reply:
Yup yup, I figured. Fixed now